bin/init.sh

```diff
@@ -21,15 +21,7 @@ if [ ${ROOT} != "${GO_TOP:-$HOME/go}/src/istio.io/istio" ]; then
        exit 1
 fi
 
-DEP=$(which dep || echo "${GO_TOP}/bin/dep" )
-echo "using dep: ${DEP}"
-
-if [ ${DEP} == "${GO_TOP}/bin/dep" ]; then
-    unset GOOS && go get -u github.com/golang/dep/cmd/dep
-fi
-
```

Comment:
why remove these?

Reply:
Seems like he's changed it to call the init script with the DEP env var?

Reply:
The makefile is doing it already, and this was causing a 'go get' on every make ( even if the dep was already there ). The original fix was a fix for mac, where dep was compiled with GOOS=linux